package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Mock STDIN file descriptor in Node.js",
   "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
```

Comment:
I think we still need to update this, right?

Reply:
pffft yeah silly me!

Reply:
I meant, to point to the file in the root directory. I could be wrong, not an expert in the typescript stuff, so if it’s fine lgtm

Reply:
> I could be wrong

You're not - that was my literal reaction when I saw your comment & the line, b/c that was the point of the PR in the first place and so couldn't believe I'd forgotten to update it 😹 

I've amended my commit to change that, so it's fixed now; good catch 😄